core/contracts/oracle/implementation/DesignatedVoting.sol

```diff
@@ -5,7 +5,7 @@ pragma experimental ABIEncoderV2;
 import "../../common/MultiRole.sol";
 import "../../common/Withdrawable.sol";
 import "../interfaces/VotingInterface.sol";
-import "../../Finder.sol";
+import "../interfaces/FinderInterface.sol";
 import "./Voting.sol";
```

Comment:
Probably outside the scope of this PR, but does this contract need to import both Voting and Voting Interface?

Reply:
Nope, but since it uses the batch methods, we either need to move those methods to the interface or rely on Voting directly.